src/main.ts

```diff
@@ -93,9 +119,24 @@ const command: Command = {
 		});
 	},
 	run(helper: Helper, args: TestArgs) {
+		/* istanbul ignore next */
+		function unhandledRejection(reason: any) {
```

Comment:
*in an upcoming release* is not inherently currently available...  Given it is highly likely to be forwards compatible.

A test case makes sense.